src/inputs/Table/components/ImportButton/ImportButton.js

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import UploadFileIcon from '@mui/icons-material/UploadFile';
+import { Tooltip, IconButton } from '@mui/material';
+import PropTypes from 'prop-types';
+
+export const ImportButton = (props) => {
+  const { editMode, onImport, label } = props;
+  return (
+    <Tooltip title={label}>
+      <span>
+        <IconButton
+          variant="text"
+          component="label"
+          onChange={onImport}
+          size="small"
+          color="primary"
+          disabled={!editMode}
+          data-cy="import-file-button"
+        >
+          <input type="file" accept=".csv, .xlsx" hidden />
+          <UploadFileIcon fontSize="inherit" />
+        </IconButton>
+      </span>
+    </Tooltip>
+  );
+};
+
+ImportButton.propTypes = {
+  /*
+   * Function use to import a file in Table component readable by AgGrid
+   */
+  onImport: PropTypes.func,
```

Comment:
`onImport` should either be required, or the component should return `null` when its value is not defined